### PR TITLE
add account creation for a user.

### PR DESCRIPTION
**Pull request by bfpepper**
Hey guys. Here is the test for account creation. There is no styling yet but the user model is made along with necessary migrations. Please merge if it looks good. 

@bermannoah @antciccone @lsaville

### CODE REVIEW

**Comment by bfpepper** [comment]
Closes #10

File: Gemfile [hunk 1]
```diff
@@ -39,16 +23,16 @@ group :development, :test do
   gem 'launchy'
   gem 'database_cleaner'
   gem 'byebug'
-  gem 'pry'
+  gem 'pry-rails'
   gem 'vcr'
   gem 'simplecov'
+  gem 'webmock'
```
Comment (bermannoah) [inline Gemfile]: why is this in development, doesn't that blow things up?

File: spec/features/a_visitor_can_create_an_account_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    visit new_user_path
+
+    fill_in 'user[name]', with: "Test Bob"
+    fill_in "user[phone]", with: "303-555-5555"
+    fill_in "user[email]", with: "test_email@example.com"
+    fill_in "user[email_confirmation]", with: "test_email@example.com"
+    fill_in "user[password]", with: "password"
+    fill_in "user[password_confirmation]", with: "password"
+
```
Comment (bermannoah) [inline spec/features/a_visitor_can_create_an_account_spec.rb:14]: why not use awesome faker gem stuff here?

File: Gemfile [hunk 3]
```diff
@@ -24,10 +24,12 @@ group :development, :test do
...
   gem 'pry-rails'
-  gem 'vcr'
-  gem 'simplecov'
-  gem 'webmock'
+end
 
+group :test do
+  gem 'webmock'
+  gem 'simplecov'
+  gem 'vcr'
```
Comment (bermannoah) [inline Gemfile:32]: yay, thanks!

**Comment by bermannoah** [comment]
looks good to merge! 👍

**Pull request merged**